modules/spacelift/variables.tf

```diff
@@ -41,3 +64,134 @@ variable "terraform_version_map" {
   description = "A map to determine which Terraform patch version to use for each minor version"
   default     = {}
 }
+
+variable "stack_config_path_template" {
+  type        = string
+  description = "Stack config path template"
+  default     = "stacks/%s.yaml"
+}
+
+variable "policies_available" {
+  type        = any
+  description = "List of available default policies to create in Spacelift (these policies will not be attached to Spacelift stacks by default, use `var.policies_enabled`)"
+  default = [
+    "git_push.proposed-run",
+    "git_push.tracked-run",
+    "plan.default",
+    "trigger.dependencies",
+    "trigger.retries"
+  ]
+}
+
+variable "policies_enabled" {
+  type        = any
```

Comment:
please change all any's to their respective type.

we had a customer complaint about this recently.